src/main/java/racingcar/validator/Validator.java

```diff
@@ -0,0 +1,40 @@
+package racingcar.validator;
+
+public class Validator {
+
+    public void carNameLengthValidate(String car) {
+
+        if(car.length() > 5) {
+            throw new IllegalArgumentException("레이싱카 이름의 길이는 5글자가 넘을 수 없습니다.");
+        }
```

Comment:
숫자 5도 상수화해서 더 의미있게 표현할 수 있을 것 같아요 !

```suggestion
        if(car.length() > CAR_NAME_MAX_LENGTH) {
            throw new IllegalArgumentException("레이싱카 이름의 길이는 5글자가 넘을 수 없습니다.");
        }
```